src/pages/[id].page.jsx

```diff
@@ -1,14 +1,12 @@
-import { useRouter } from 'next/router';
 import Layout from '../components/Layout';
+import getComponentDetails from '../../utils/airtable';
 import SpecificationBlock from '../components/SpecificationBlock';
 
-export default function Details() {
-  const router = useRouter();
-  const { id } = router.query;
+export default function Details({ componentName }) {
   return (
-    <Layout pageTitle={id}>
+    <Layout pageTitle={componentName}>
       <div>
-        <h1>{ id }</h1>
+        <h1 data-testid="heading-1">{componentName}</h1>
```

Comment:
Are we using this `testid` anywhere? I think we can remove it, and any tests looking for this element can look for specific text or an `h1` tag anyway.

Reply:
The `[id].page.test` was deleted, and then I removed that `throw error` comment, this `data-testid` that wasn't used. 
https://github.com/sparkbox/accessible-components/pull/34/commits/d6ebff5b80570b46f2d94f3ef5c124e08f070993